buffer/src/test/java/io/netty/buffer/ByteBufUtilTest.java

```diff
@@ -187,4 +187,17 @@ private static void testDecodeString(String text, Charset charset) {
         Assert.assertEquals(text, ByteBufUtil.decodeString(buffer, 0, buffer.readableBytes(), charset));
         buffer.release();
     }
+
+    @Test
+    public void testIssue4679() {
```

Comment:
Can you use a more descriptive test name? something like `testToStringDoesNotThrowIndexOutOfBounds`